### ISSUE TITLE
Missing overview of commands and shortcuts

### ISSUE THREAD

**Issue by deeplook** [jonathanslenders/pymux#70]
This is a great project! Thanks!

One comment I have is the following. The current README says about pymux vs tmux:

> To some extent, pymux is a clone of tmux. This means that all the default shortcuts are the same; the commands are the same or very similar, and even a simple configuration file could be the same. (There are some small incompatibilities.) However, we definitely don't intend to create a fully compatible clone. Right now, only a subset of the command options that tmux provides are supported.

It would be helpful to have an overview of the commands actually supported by pymux. Otherwise people excited to use pymux must go learn about tmux first to find out its commands and will still not know in detail if pymux supports the same or not. This popular "tmux shortcuts & cheatsheet" might serve as an inspiration: https://gist.github.com/MohamedAlaa/2961058

One might even include such an overview into pymux itself, in some manner similar to what Jupyter does (which should not be a great challenge using the prompt_toolkit). See image (a part of the full info) below. See also this PDF: https://conferences.oreilly.com/jupyter/jup-ny/public/content/jupyter-shortcuts?download=true

![screen shot 2017-12-03 at 12 01 13](https://user-images.githubusercontent.com/1001778/33524734-8c02a7f4-d822-11e7-89f2-dc5a79897825.png)

**Comment by jonathanslenders** [comment]
Agreed. So far I didn't had any time yet, but this would be good to have indeed.
